### DISCUSSION TITLE
brotli support

### DISCUSSION THREAD

**Discussion by szuecs** [klauspost/compress#671] [Ideas/Features]
https://github.com/klauspost/compress/issues/9 was closed by the issue creator and it seems that there is a working library that supports reader and writer for brotli https://github.com/andybalholm/brotli. 

Any blockers in order to have this in this library, too?

**Comment by klauspost** [comment]
@szuecs Not as such, but I will likely have to do a complete rewrite for it to make sense to have here.

I don't just want to copy+paste code in here to have it here.
